catalog/app/components/Preview/loaders/useGate.ts

```diff
@@ -0,0 +1,79 @@
+import type { S3, AWSError } from 'aws-sdk'
+
+import type * as Model from 'model'
+import * as AWS from 'utils/AWS'
+import * as Data from 'utils/Data'
+import log from 'utils/Logging'
+
+import { PreviewError } from '../types'
+
+interface SizeThresholds {
+  autoFetch: number
+  neverFetch: number
+}
+
+const SIZE_THRESHOLDS: SizeThresholds = {
+  autoFetch: 128 * 1024, // automatically load if <= 128kB
+  neverFetch: 1024 * 1024, // never load if > 1MB
```

Comment:
```suggestion
  neverFetch: 1024 * 1024, // never load if > 1MiB
```

it's time to [stop abusing SI prefixes](http://stopabusingsiprefixes.org/stopabusing.html) =)